pkg/upgrade/control_plane.go

```diff
@@ -638,14 +606,17 @@ func (u *ControlPlaneUpgrader) shouldSkipMachine(log logr.Logger, machine *clust
 		return true
 	}
 
+	// Skip if this machine doesn't have NameBase annotation
 	baseName := machine.Annotations[AnnotationMachineNameBase]
 	if baseName == "" {
-		// should never happen
+		// should never happen because we call reconcileMachineAnnotations for all
+		// machines in the beginning.
 		log.Error(errors.New("machine is missing annotation "+AnnotationMachineNameBase), "unable to upgrade machine")
```

Comment:
@ncdc Should this be logged as an `Error` or as `Info`? Even the previous condition which seems just as pertinent is only logged out as `Info`.

Reply:
This should be an Info log.

My reasoning, and this does get somewhat subjective, is because this code handles this case and returns a valid value that the caller knows what to do with. Code execution is not stopped and therefore this isn't an error condition.

Reply:
Thanks. I was leaning towards that as well.